lib/trino-hdfs/src/main/java/io/trino/filesystem/hdfs/HdfsTrinoInputStream.java

```diff
@@ -107,6 +107,22 @@ public int read(byte[] b, int off, int len)
         }
     }
 
+    @Override
+    public long skip(long n)
+            throws IOException
+    {
+        ensureOpen();
+        try {
+            return stream.skip(n);
+        }
+        catch (FileNotFoundException e) {
```

Comment:
I think we can remove the `FileNotFoundException` block, since skip shouldn't be triggering the first attempt to open the resource (ideally).